surfsense_backend/app/agents/researcher/qna_agent/nodes.py

```
@@ -87,55 +88,59 @@ async def answer_question(state: State, config: RunnableConfig) -> Dict[str, Any
-            </metadata>
-            <content>
-                {content}
-            </content>
-        </document>
+        # Create the formatted documents text
+        documents_text = f"""
+        Source material from your personal knowledge base:
+        <documents>
+            {"\n".join(formatted_documents)}
+        </documents>
         """
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string expression part cannot include a backslash (app.agents.researcher.qna_agent.nodes, line 124)'

(E0001)